src/components/organisms/DashboardContent/DashboardContent.tsx

```diff
@@ -138,29 +141,43 @@ const DashboardContent: React.FC<any> = props => {
       return;
     }
 
-    if (contentProps.data === null || contentProps.data?.results === []) {
-      dispatch(setData([]));
-
-      if (canSelectRow) {
-        dispatch(setSelectedRecord({selectedRecord: null}));
+    if (contentProps.data || contentProps.data?.length) {
```

Comment:
You can maybe create a condition variable and pass it to only one `if`. There is unnecessary to put 3 `ifs` in a row without `else`

Reply:
Resolved!